spec/e2e/tillReceiptFeature.js

```diff
@@ -0,0 +1,20 @@
+describe('Till receipt', function() {
+
+  beforeEach(function() {
+    browser.get('http://localhost:8080');
+  });
+
+  it('should have the app name on the page', function() {
+  });
+
+  it('should have the menu on the page', function() {
+  });
+
+  it('should let the cashier enter the item and quantity', function() {
+  	element(by.repeater("(key, value) in mainCtrl.items")).click("Cappuccino")
```

Comment:
Mixed double and single quotes.<br>Missing semicolon.<br>'element' is not defined.<br>'by' is not defined.